### ISSUE TITLE
Kivent-Maps example 14 terminates with 'integer required'

### ISSUE THREAD

**Issue by jeffersonheard** [kivy/kivent#196]
Gist of kivy console: https://gist.github.com/jeffersonheard/996dd07872167a8a84651e716a8cbcd2

I just installed kivy from pip and then checked out and built the core and maps modules with kivent. Here is the result of a `pip freeze`

```
cymunk==0.0.0
Cython==0.24.1
KivEnt-Core==2.2.0.dev0
KivEnt-maps==1.0.0
Kivy==1.9.1
Kivy-Garden==0.1.4
pathlib==1.0.1
requests==2.12.5
six==1.10.0
tmx==1.9.1
```

Platform: OS X 10.11.6
Python: 2.7.11 (system python build)

**Comment by spinningD20** [comment]
downgrade tmx to 1.8.1 and test

**Comment by jeffersonheard** [comment]
Yes, that works to bring it up, but I now get the same logging issue I do in #197 . 

`[ERROR  ] OpenGL Error (Renderer._set_blend_func-preglBlendFunc) 1282 / 502 for None`
